### ISSUE TITLE
Does not support list, dict as values

### ISSUE THREAD

**Issue by anroopak** [indietyp/django-automated-logging#6]
I tried adding DAL to my project. Some of the models using ArrayField, JSONField using postgres lib. 
For such models the audit log is not saved and errors out with the following exception. 

```
File "/xxxxx/automated_logging/signals/save.py", line 97, in pre_save_signal
    for k, v in old.items()
TypeError: unhashable type: 'list'
```

**Comment by indietyp** [comment]
Thank you very much for the bug report, the tests are using SQLite as the database so this must've slipped by.

Investigating the issue.

**Comment by indietyp** [comment]
fixed in the 6.1.1 release
